src/package-modules.js

```diff
@@ -548,10 +550,12 @@ async function createMetaPackageFromRepoDir(url, dir, ref, options) {
   composerConfig.version = version;
   setDependencyVersions(composerConfig, dependencyVersions);
 
+  const finalComposerConfig = (transform && transform[name] || []).reduce((config, transformFn) => transformFn(config), composerConfig);
```

Comment:
You could change this to a `let composerConfig` I don't see a great reason for two separate variables.

Reply:
Thanks, done. Jjust running a test locally, then I'll update the PR.